openfl/component/aggregation_functions/weighted_average.py

```diff
@@ -2,21 +2,25 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """Federated averaging module."""
+from typing import Any
+from typing import List
 
 import numpy as np
+from numpy.typing import ArrayLike
 
+from openfl.utilities import LocalTensor
 from .core import AggregationFunction
 
 
-def weighted_average(tensors, weights):
+def weighted_average(tensors: ArrayLike, weights: ArrayLike) -> np.ndarray:
```

Comment:
`ArrayLike` because `tensors` is `Tuple[np.ndarray]` and `weights` is `Tuple[np.ndarray]` and `np.average(...)` take's `ArrayLike` [type](https://numpy.org/devdocs/reference/typing.html#numpy.typing.ArrayLike).
We discussed it [here](https://github.com/intel/openfl/pull/377#discussion_r829803101)